<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OwnTupleView</class>
 <widget class="QMainWindow" name="OwnTupleView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>313</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>350</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Bitte geben Sie hier das gewünschte Zahlentupel ein:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutValues"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutLabels"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutValues2"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutLabels2"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnFinish">
      <property name="text">
       <string>Abschließen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
